File: Data Structures & Algorithms/concatenation-of-array/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number[]}
     */
    getConcatenation(nums) {
        //arr nums length n

        const arrLength = nums.length * 2;
        const newArray = new Array(arrLength).fill(0);

        for(let i = 0; i < nums.length; i++){
            newArray[i] = nums[i];
            newArray[i + nums.length] = nums[i];
        }

        return newArray;

    }
}

File: Data Structures & Algorithms/replace-elements-with-greatest-element-on-right-side/submission-1.py
class Solution:
    def replaceElements(self, arr: List[int]) -> List[int]:
        if len(arr) == 1:
            return [-1]

        for i in range(len(arr)):
            if(i == len(arr) - 1):
                arr[i] = -1
            greatest = -1
            for j in range(i+1, len(arr)):
                if(arr[j] > greatest):
                    greatest = arr[j]
            arr[i] = greatest
        return arr



            